src/eduid_idp/assurance.py

```diff
@@ -141,7 +154,7 @@ def response_authn(req_authn_ctx, user, sso_session, logger):
     :rtype: str | None
     """
     authn = AuthnState(user, sso_session, logger)
-    logger.debug('Authn will be evaluated based on: {!s}'.format(authn))
+    logger.info(f'Authn for user {user.eppn} will be evaluated based on: {authn}')
```

Comment:
for User, we have a nice __str__ meant to be used in logging. unless that is not true for IdPUser, suggest logging as 'Authn for {user} will be ...'